apps/notifications/src/panel/templates/summary-in-single.jsx

```diff
@@ -11,7 +14,9 @@ const Snippet = ( { note, snippet, url } ) => (
 class UserHeader extends Component {
 	render() {
 		const grav = this.props.user.media[ 0 ];
-		const grav_tag = <img src={ grav.url } height={ grav.height } width={ grav.width } />;
+		const grav_tag = (
+			<img src={ grav.url } height={ grav.height } width={ grav.width } alt="gravatar" />
```

Comment:
hmmmm, should "gravatar" be translated or is it just a company name that's the same in all languages?